src/Microsoft.Tye.Core/GenerateKubernetesManifestStep.cs

```diff
@@ -12,6 +12,8 @@ public sealed class GenerateKubernetesManifestStep : ServiceExecutor.Step
 
         public string Environment { get; set; } = "production";
 
+        public string Namespace { get; set; } = "default";
```

Comment:
I think a better default for this property would be `null`. Currently we use the namespace of the current context (different from being hardcoded to `default`).